test/evergreen.yml

```diff
@@ -2386,6 +2386,24 @@ tasks:
           times: 1       # No of times to run the loop
           no_of_procs: 10 # No of processes to run in the background
 
+  - name: performance-test
```

Comment:
Can we use a more specific task name, e.g. `many-handle-stress-test`?

Reply:
Sure, I will rename it `many-dhandle-stress-test` to reflect the python script used.